.github/workflows/dokken-validate.yml

```diff
@@ -23,9 +23,19 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@main
+      - name: Get changed files
+        id: changed-files-excluding-tests
+        uses: tj-actions/changed-files@v35
+        with:
+          files_ignore: |
+            !.github
+            !**/aws-parallelcluster-*/spec
```

Comment:
If I understood correctly we're skipping to execute the validate if there are changes in the test code.

This means that if the tests are broken and we're adding a commit to fix them they won't be executed, so won't have confirmation about the fix. Am I correct?

Reply:
Correct, if the fix contains only the test code. GH Action is required to certify changes to production code: you can always test changes to your test locally.
Furthermore, we can change these conditions in the future if we find out they don't cover correctly some corner cases.